packages/@lwc/integration-karma/test/directive-external/index.spec.js

```diff
@@ -0,0 +1,76 @@
+import { createElement } from 'lwc';
```

Comment:
I put the basic tests in `packages/@lwc/integration-karma/test/template/directive-external/` in https://github.com/salesforce/lwc/pull/3144. Maybe we can put this test there too under a sub-directory?

Reply:
Let's get your PR merged first and I'll deal with any conflicts after.